### ISSUE TITLE
render recipe for use with planetscale

### ISSUE THREAD

**Issue by Vandivier** [blitz-js/blitz#3053]
### What do you want and why?

As a developer, I would like to be able to easily integrate planetscale db, blitz, and render.com

### Possible implementation(s)

1. `blitz install render` supports a planetscale-compatible YAML
2. `blitz install render-planetscaledb` or similar; basically a new recipe
3. no recipe, just update [deploy to render docs](https://blitzjs.com/docs/deploy-render) to include YAML that works with planetscale db.

**Comment by Vandivier** [comment]
on second thought maybe this should be a project template instead of a recipe? rationale: currently when I `npx blitz@latest new myApp` eventually it will tell me something like:
```
Success! Next steps:
cd myApp
blitz prisma migrate dev
blitz dev
```
however I think we shouldn't migrate with planetscale, so in theory users could learn to "just ignore that" but this isn't really ergonomic

**Comment by Vandivier** [comment]
tangentially related:
https://github.com/blitz-js/blitz/discussions/3058#discussioncomment-1791049

**Comment by Vandivier** [comment]
another pain point is that the `db.$reset` helper seems to fail when connected to planetscale, maybe bc it calls migrate underneath

![image](https://user-images.githubusercontent.com/5559355/145691110-a6855e06-64d4-4925-8d5a-1a90ca6760e3.png)

**Comment by beerose** [comment]
> on second thought maybe this should be a project template instead of a recipe? rationale: currently when I `npx blitz@latest new myApp` eventually it will tell me something like:
> 
> ```
> Success! Next steps:
> cd myApp
> blitz prisma migrate dev
> blitz dev
> ```
> 
> however I think we shouldn't migrate with planetscale, so in theory users could learn to "just ignore that" but this isn't really ergonomic

I think the recipe makes sense. If user creates a new Blitz app, then can first play with it with Sqlite, and then install planetscale recipe. This recipe should remove the `migrations` folder and do all the required changes in the `db` directory. It can also remove references to `$reset` if needed or replace it with something that works with planetscale. What do you think>

**Comment by Vandivier** [comment]
Makes sense to me! I guess that's the same pattern we follow for any remote db (postgres, etc) currently anyway
